CODE_OF_CONDUCT.md

```diff
@@ -1,80 +1,127 @@
-# Contributor Covenant Code of Conduct
+# VOLK Code of Conduct
 
-This Code of Conduct is also available [here on the GNU Radio
-wiki](https://wiki.gnuradio.org/index.php/Code_of_Conduct).
+This Code of Conduct is adapted from the GNU Radio Code of Conduct,
+available in the [gr-governance repository][GRCoC], dated 2021-07-12.
 
 ## Our Pledge
 
-In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to making participation in our project and
-our community a harassment-free experience for everyone, regardless of age, body
-size, disability, ethnicity, sex characteristics, gender identity and expression,
-level of experience, education, socio-economic status, nationality, personal
-appearance, race, religion, or sexual identity and orientation.
+We as members, contributors, and leaders strive to make participation
+in our community a harassment-free experience for
+everyone. Period. Qualities and characteristics we note specifically
+include regardless of age, body size, visible or invisible disability,
+ethnicity, sex characteristics, gender identity, gender expression,
+sexual identity, sexual orientation, level of experience, education,
+socio-economic status, nationality, personal appearance, race, caste,
+color, religion, and military status -- but lists like these cannot be
+complete and hence we note "for everyone. Period." in order to be as
+inclusive as possible.
```

Comment:
I assume this is the paragraph where you added some things?
I'd say the goal is clear. It sounds right. Only, I'd remove the "Period." parts. These make it sound grumpy.

Reply:
OK & That's fine with me. Here is [the original GR source](https://github.com/gnuradio/gr-governance/blob/main/CODE_OF_CONDUCT.md).